### ISSUE TITLE
Ethernet description not shown in show interface ethernetX

### ISSUE THREAD

**Issue by mingjunzhang2019** [project-arlo/sonic-mgmt-framework#89]
Z9100, Build label:299:

```
sonic(conf-if-Ethernet4)# description "DelSonic"
Success
sonic(conf-if-Ethernet4)# do show interface status
------------------------------------------------------------------------------------------
Name                Description         Admin          Oper           Speed          MTU
------------------------------------------------------------------------------------------
Ethernet0           -                   down           down                          9100
Ethernet4           DelSonic            down           down                          9100

sonic(conf-if-Ethernet4)# do show interface Ethernet 4
Ethernet4 is down, line protocol is down
Hardware is Eth
Interface index is 2
Mode of IPV4 address assignment: not-set
Mode of IPV6 address assignment: not-set
IP MTU 9100 bytes
LineSpeed , Auto-negotiation off
Input statistics:
        0 packets, 0 octets
        0 Multicasts, 0 Broadcasts, 0 Unicasts
        0 error, 0 discarded
Output statistics:
        0 packets, 0 octets
        0 Multicasts, 0 Broadcasts, 0 Unicasts
        0 error, 0 discarded
sonic(conf-if-Ethernet4)#                      
```

**Comment by mingjunzhang2019** [comment]
`no description` not work:

```
sonic(conf-if-Ethernet4)# no description
%Error: Invalid value
sonic(conf-if-Ethernet4)#          
```

**Comment by justinejose91** [comment]
I will look into it.

**Comment by justinejose91** [comment]
Description not present in #show interface ethernet and with id is fixed. #no description command issue is still open.

**Comment by justinejose91** [comment]
When we do #no description command for an interface, when I set the description to empty string and set operation, CVL throws failure. Added the CVL logs below.
[cvl_logs.txt](https://github.com/project-arlo/sonic-mgmt-framework/files/3592676/cvl_logs.txt)
When I tried removing the description entry from PORT table, then CVL doesnt throw failure, but app-db is not getting reflected.
Rest Request = {"openconfig-interfaces:description": ""} for any physical interface.

**Comment by jeffyin-dell** [comment]
@sachin-brm @dutta-partha -- can someone help look into the CVL failure when setting description empty string? This had previously worked in our testing.

**Comment by dutta-partha** [comment]
@asha-behera Please check the CVL error log once. 

@justinejose91 Are you able to reproduce this every time ?

**Comment by justinejose91** [comment]
@dutta-partha Yes, this is reproducible every time. I tried deleting description entry as an other option, that won't reflect it in APP-DB, but setting it as empty string used to work earlier, now CVL throws error.

**Comment by dutta-partha** [comment]
@justinejose91 CVL fix for resetting description configuration is available in master branch (PR #118). @justinejose91 Can you please check it.

**Comment by justinejose91** [comment]
Fixed. Works fine.
